src/Commands/Connection/InfoCommand.php

```diff
@@ -21,67 +21,48 @@ class InfoCommand extends TerminusCommand
      *
      * @command connection:info
      *
+     * @field-labels
+     *   sftp_username: SFTP Username
+     *   sftp_host: SFTP Host
+     *   sftp_password: SFTP Password
+     *   sftp_url: SFTP URL
+     *   sftp_command: SFTP Command
+     *   git_username: Git Username
+     *   git_host: Git Host
+     *   git_port: Git Port
+     *   git_url: Git URL
+     *   git_command: Git Command
+     *   mysql_host: Mysql Host
+     *   mysql_username: Mysql Username
+     *   mysql_password: Mysql Password
+     *   mysql_port: Mysql Port
+     *   mysql_database: Mysql Database
+     *   mysql_url: Mysql URL
+     *   mysql_command: Mysql Command
```

Comment:
About the capitalization of the field labels, the current version of Terminus output looks like this:
```
+----------------+------------------------------------------------------------+
| Key            | Value                                                      |
+----------------+------------------------------------------------------------+
| Sftp Username  | dev.ac2022fe-4223-4e80-a290-42a0db87a1d6                   |
| Sftp Host      | appserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush.i |
|                | n                                                          |
| Sftp Password  | Use your account password                                  |
| Sftp Url       | sftp://dev.ac2022fe-4223-4e80-a290-42a0db87a1d6@appserver. |
|                | dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush.in:2222     |
| Sftp Command   | sftp -o Port=2222 dev.ac2022fe-4223-4e80-a290-42a0db87a1d6 |
|                | @appserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush. |
|                | in                                                         |
| Git Username   | codeserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6        |
| Git Host       | codeserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush. |
|                | in                                                         |
| Git Port       | 2222                                                       |
| Git Url        | ssh://codeserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6@ |
|                | codeserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush. |
|                | in:2222/~/repository.git                                   |
| Git Command    | git clone ssh://codeserver.dev.ac2022fe-4223-4e80-a290-42a |
|                | 0db87a1d6@codeserver.dev.ac2022fe-4223-4e80-a290-42a0db87a |
|                | 1d6.drush.in:2222/~/repository.git wp-update               |
| Mysql Host     | dbserver.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush.in |
| Mysql Username | pantheon                                                   |
| Mysql Password | 4b33cbdfd64c446788bef86bf9f3af6c                           |
| Mysql Port     | 16191                                                      |
| Mysql Database | pantheon                                                   |
| Mysql Url      | mysql://pantheon:4b33cbdfd64c446788bef86bf9f3af6c@dbserver |
|                | .dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush.in:16191/p |
|                | antheon                                                    |
| Mysql Command  | mysql -u pantheon -p4b33cbdfd64c446788bef86bf9f3af6c -h db |
|                | server.dev.ac2022fe-4223-4e80-a290-42a0db87a1d6.drush.in - |
|                | P 16191 pantheon                                           |
+----------------+------------------------------------------------------------+
```
I see that I haven't capitalized things exactly the same as it previously was either. Most places use "URL" rather than "URL", and "MySQL" as you wrote it.

@ari-gold I guess a change in capitalization here is appropriate, i.e. from "Mysql Url" to "MySQL URL"? I know @ajbarry said you wanted to just to use mysql_url here, but that isn't necessarily required simply for the purpose of field selection. Regardless, we should decide and standardize.